substrate/frame/revive/uapi/src/host.rs

```diff
@@ -63,6 +63,14 @@ pub trait HostFn: private::Sealed {
 	/// - `output`: A reference to the output data buffer to write the balance.
 	fn balance(output: &mut [u8; 32]);
 
+	/// Stores the *free* balance of the supplied address into the supplied buffer.
```

Comment:
I think we should talk about reducible balance here which is free balance minus the existential deposit.

Reply:
I agree